samtranslator/model/api/api_generator.py

```diff
@@ -1224,9 +1225,11 @@ def _set_default_authorizer(
                 add_default_auth_to_preflight=add_default_auth_to_preflight,
             )
 
-    def _set_default_apikey_required(self, swagger_editor: SwaggerEditor) -> None:
+    def _set_default_apikey_required(
+        self, swagger_editor: SwaggerEditor, AddApiKeyRequiredToCorsPreflight: bool
```

Comment:
Please rename `AddApiKeyRequiredToCorsPreflight` to use snake_case naming stype.